docs/pages/reference/session-recording.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Teleport Session Recording
+description: An overview of Teleport's session recording and its configuration
+---
+
+This page provides an overview of how Teleport records sessions and the various
+configuration options that apply to session recordings.
+
+Teleport supports two different recording configurations: synchronous and
```

Comment:
Ack. I'm being very careful to say "configurations" and not "recording modes" here, because unfortunately some recent work has stolen the term "recording modes" and I don't want to go overboard in this doc.